ddsrouter_core/include/ddsrouter_core/configuration/participant/SimpleParticipantConfiguration.hpp

```diff
@@ -60,7 +60,7 @@ class SimpleParticipantConfiguration : public ParticipantConfiguration
 
     types::DomainId domain_;
 
-    static const types::DomainId DEFAULT_DOMAIN_ID_; // 0
+    DDSROUTER_CORE_DllAPI static const types::DomainId DEFAULT_DOMAIN_ID_; // 0
```

Comment:
We should avoid exporting protected members or functions. I don't see why you need this.

Reply:
You are probably right, but it is the same situation that with `DEFAULT_DS_DOMAIN_ID_`